### PR TITLE
Feature/218 finalize landing page

### PR DESCRIPTION
**Pull request by absurd-turtle**
## Basics

- [x] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [x] The buildserver is happy.
- [x] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [x] I fully described what my PR does in the documentation
      (not in the PR description)
- [x] I fixed all affected documentation
- [x] I fixed all affected decisions
- [x] I added code comments, logging, and assertions as appropriate
- [x] I mentioned [every code or binary](/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is conforming to [our Documentation Guidelines](/doc/documentation.md)
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to our Coding Guidelines
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

### CODE REVIEW

**Comment by absurd-turtle** [comment]
jenkins build please

**Comment by markus2330** [comment]
Great job, we really like it!

- [ ] Please remove the dep to dummyimage.com (shown in Firefox uMatrix)
- [ ] Please also add Vienna, Gleisdorf, Podpec and the (case study) location of everyone else in our team (@badnames @aidnurs @Bushuo) in the map
- [ ] make gallery as large as other parts of website
- [ ] Logo should be with colors, for our first ideas see Logo/illustrationen_samuel/permaplant-logo-2-colors.*
- [ ] Please add an imprint page
- [ ] small logo should be top left

**Comment by markus2330** [comment]
And please readd authors with everyone in doc/authors.md and Dr Pavlo Ardanov, NGO "Permaculture in Ukraine".

**Comment by Bushuo** [comment]
My case study will be done at 1210 Wien, Strebersdorferstraße (48.2863480, 16.3979439)

**Comment by markus2330** [comment]
Excellent job, we love it!

- [ ] small logo in light mode is broken
- [ ] "type writer" text should stop after one loop
- [ ] unify green color (logo, small images, leaf), see #194 
- [ ] imprint: Markus Raab and Yvonne Markl with same contact form as in landing page
- [ ] Ramzan M. -> Magomadow (@NurramoX if it is okay for you)
- [ ] Markus Raab -> Dr. Markus Raab
- [ ] Yvonne Markl -> Yvonne Markl, MSc.
- [ ] add Gallicano, Toscana; Feldthurns; and Kufstein to map
- [ ] gallery should be small by default
- [ ] instead of "1.2K 6", add date; remove "Learn More"
- [ ] dark mode by default (after browser start)
- [ ] readd pricing

**Comment by absurd-turtle** [comment]
>     * [x]  dark mode by default (after browser start)
The dark mode is enabled as default when the user has never visited the page. You can try it in private window :)

**Review by markus2330 — APPROVED** [review]

**Comment by markus2330** [comment]
@Bushuo should we merge this or #235 first?

**Comment by Bushuo** [comment]
I think merging this first is better. Then additional issues for the translations can be made after #235 is merged and this will stay relatively small.

**Review by Bushuo — COMMENTED** [review]
Very nice :)

File: frontend/src/components/Layout/Navbar.tsx [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
       <div className="mx-auto flex max-w-screen-xl flex-wrap items-center justify-between p-4">
         <div className="flex space-x-16">
           <div className="flex space-x-6">
             <a href="/" className="flex items-center">
+              {darkMode ? (
+                <img
+                  src="/permaplant-logo.svg"
+                  className="w-12 pr-2"
+                  alt="permaplant logo small"
+                ></img>
```
Comment (buenaflor) [inline frontend/src/components/Layout/Navbar.tsx:23]: m: you could use the new vite plugin to load in svgs as react component, see features: https://github.com/ElektraInitiative/PermaplanT/blob/826466685cfef4df0ecd1b3bab68c2d4b43e9ad4/frontend/src/features/landing_page/components/Features.tsx#L3

File: frontend/public/permaplant-logo-2-colors.svg
[the remark this answers is not in the document]
Comment (absurd-turtle) [reply frontend/public/permaplant-logo-2-colors.svg:1]: Yes, this was on purpose

**Comment by absurd-turtle** [comment]
@markus2330 Can we merge this?

**Comment by markus2330** [comment]
Thank you, looking forward to see the changes!

**Pull request merged**